components/org.apache.stratos.autoscaler/src/main/java/org/apache/stratos/autoscaler/applications/ApplicationUtils.java

```diff
@@ -191,52 +128,56 @@ public static StringBuilder getTextPayload (String appId, String groupName, Stri
         return payloadBuilder;
     }
 
-    public static PayloadData createPayload(String appId, String groupName, CartridgeInfo cartridgeInfo, String subscriptionKey, int tenantId, String clusterId,
+    public static PayloadData createPayload(String appId, String groupName, Cartridge cartridge, String subscriptionKey, int tenantId, String clusterId,
```

Comment:
Yes indeed, we already have a bean class for these parameters and it is PayloadData. Since createPayload is the method that executes this logic I don't think it would be appropriate to introduce another bean class in this scenario.